packages/react/src/components/tag.tsx

```diff
@@ -30,6 +34,17 @@ export interface TagProps extends HTMLAttributes<HTMLElement> {
    */
   variant?: TagVariant
 
+  /**
+   * What should the tag severity be? Affects the color of the tag.
+   */
+  severity: TagSeverity
+
+  /**
+   * What icon should be used?
+   * @default null
+   */
+  icon?: React.ReactNode
```

Comment:
Don't use the global namespace but use explicit imports instead e.g.

```
import type { ReactNode } from 'react'
```

this is to stay consistent with the rest of the codebase